pages/application.tsx

```diff
@@ -83,185 +63,190 @@ const Application: NextPage = () => {
+										}}
+									/>
+									<NativeSelect
+										label="How many hackathons have you participated in?"
+										{...form.getInputProps('hackathonCount')}
+										data={[
+											{ value: 'N', label: 'My first hackathon!' },
+											{ value: 'S', label: '1-3' },
+											{ value: 'M', label: '4-6' },
+											{ value: 'L', label: '7+' }
+										]}
+									/>
+									{/* TODO: FileInput has a Capture prop has type boolean | "user" | "environment"
+										Add after we know hosting server and resume site api */}
+									<FileInput
+										label="Upload your resume. PDF only!"
+										description="Resume is optional, but highly encouraged for your application."
+										{...form.getInputProps('resume')}
+										placeholder="Submit resume"
+										accept="application/pdf"
+									/>
+									<TextInput label="LinkedIn" {...form.getInputProps('linkedin')} />
+									<TextInput label="GitHub" {...form.getInputProps('github')} />
+									<MultiSelect
+										label="Other Sites"
+										{...form.getInputProps('otherSites')}
+										data={otherURLs}
+										searchable
+										creatable
+										getCreateLabel={(query) => query}
+										onCreate={(query) => {
+											setOtherURLs([...form.values.otherSites, query]);
+											return query;
+										}}
+										rightSection={null}
+									/>
+									<Switch label="I am looking for a team" {...form.getInputProps('lookingForTeam')} />
+									<Switch label="I am interested in attending the PreHacks event" {...form.getInputProps('attendingPrehacks')} />
+									<Group position="right" mt="md">
+										<Button
+											variant="default"
+											onClick={() => {
+												setStep(step - 1);
+											}}>
+											Back
+										</Button>
+										<Button
+											onClick={() => {
+												if (!form.validate().hasErrors) {
+													setStep(step + 1);
+												}
+											}}>
+											Next
+										</Button>
+									</Group>
+								</Stack>
+							</Box>
+						</Stepper.Step>
+						<Stepper.Step label="Consent">
+							<Box sx={{ maxWidth: 600 }} mx="auto">
+								<Stack>
+									<Checkbox
+										required
+										label={
+											<>
+												I have read and agree to the{' '}
+												<a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank" rel="noopener noreferrer">
+													MLH Code of Conduct.
+												</a>
+											</>
+										}
+										{...form.getInputProps('codeOfConductAgreement')}
+									/>
+									<Checkbox
+										required
+										label={
+											<>
+												I authorize you to share my application/registration information for event administration, ranking, MLH
+												administration, pre- and post-event informational e-mails, and occasional messages about hackathons in-line
+												with the MLH Privacy Policy. I further I agree to the terms of both the
```

Comment:
The label for this field currently displays like this:
![image](https://user-images.githubusercontent.com/44961593/219981858-1736ab69-04c1-44af-a1a3-06fd91818886.png)
The two things to do is make the space before "MLH Contest Terms and Conditions" not part of the hyperlink, and to add a space after the "and the" preceding "MLH Privacy Policy".